concourse/steps/component_descriptor.mako

```diff
@@ -3,12 +3,18 @@
   filter="indent_func(indent),trim"
 >
 <%
+import concourse.model.traits.version
+import concourse.model.traits.component_descriptor
 from makoutil import indent_func
 from concourse.steps import step_lib
+
 descriptor_trait = job_variant.trait('component_descriptor')
 main_repo = job_variant.main_repository()
 main_repo_path_env_var = main_repo.logical_name().replace('-', '_').upper() + '_PATH'
 
+version_file_path = job_step.input(concourse.model.traits.version.ENV_VAR_NAME) + '/version'
```

Comment:
?????
use os.path.join

Reply:
also - we already had this discussion - several time. do. not. introduce this unnecessary indirection using env vars.

Reply:
First of, good catch - mistake when adjusting, probably taken over from https://github.com/gardener/cc-utils/blob/41c03c4a9454118586e597e241774265a9277b90/concourse/steps/draft_release.mako#L8

Second of, I'm going to remove the commits concerning the env-var stuff alltogether. I have stated my intent to 'remove hard-coded in/output names' several times, but considering the second remark that seems moot. I see that the pr name is misleading, however. My apologies, I'll take a closer look at GHs suggestions in the future.